index.html

```diff
@@ -67,9 +67,9 @@ <h3 class="connect-link">
                 <h2 class="card-title">Tonic</h2>
                 <ul class="position">
```

Comment:
- [ ] Keep doing the great work with the heading hierarchies :+1:

Reply:
Thanks.